ui/projects/etl-server/src/lib/pipelines/pipeline-list/pipeline-list.component.ts

```diff
@@ -47,11 +49,24 @@ export class PipelineListComponent implements OnInit {
         for (const pipeline of pipelines) {
           pipeline.display = pipeline.name;
         }
+    
         this.pipelineSections = this.processSections(pipelines, 0);
       }
     });
     this.api.queryPipelines();
   }
+  get sortBySelected() {
+    return this.sortByField;
+  }
+
+  set sortBySelected(value) {
+    this.sortByField = value;
+    if (this.sortByField !== 'last_created') {
+      this.pipelineSections.sort((a,b) => a.item[this.sortByField] > b.item[this.sortByField] ? 1 : -1);
```

Comment:
Just a reminder - you need to sort the actual pipelines and then recalculate the sections.
You might need to save the list of pipelines that we receive from the API so you can re-sort them here.